CMakeLists.txt

```diff
@@ -342,7 +341,7 @@ endif()
 if(MSVC)
     # Disable deprecation warnings about POSIX function names such as setmode (replaced by the ISO C and C++ conformant name _setmode).
     add_compile_definitions(_CRT_NONSTDC_NO_WARNINGS)
-    # Disable deprecation warnings about less secure CRT functions such as fopen (replaced by fopen_s).
+    # Disable deprecation warnings about unsafe CRT library functions such as fopen (replaced by fopen_s).
```

Comment:
The C4996 warning and these two macros are documented in https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4996?view=msvc-170